build.gradle

```diff
@@ -5,7 +5,7 @@ plugins {
 
 apply plugin: 'java'
 
-def buildNumber = System.getenv("RELEASE_VERSION")?.replace("refs/tags/", "") ?: "8.8.0-SNAPSHOT"
+def buildNumber = System.getenv("RELEASE_VERSION")?.replace("refs/tags/", "") ?: "8.7.6-HMAN-368"
```

Comment:
This should be updated to match final version number before being merged to `master`.

Reply:
yes,  This is a temporally  change.  I will be merged with the final version.

Reply:
resolved in [f6595f2](https://github.com/hmcts/befta-fw/pull/190/commits/f6595f20e4e5b51fdf0531abf0f1698dc5949411)